docs/electrolyte_database.rst

```diff
@@ -0,0 +1,59 @@
+.. _install-edb:
+
+Electrolyte Database
```

Comment:
I think this .rst file might be better off tucked away under How To and perhaps titled "How to install the Electrolyte Database".

Reply:
Moved to the "How To Guides" section.